README.md

```diff
@@ -21,6 +21,20 @@ It uses:
 ### Plugins
 * [lombok plugin](https://plugins.jetbrains.com/idea/plugin/6317-lombok-plugin) - Lombok IDEA plugin
 
+#### Environment variables
+The following environment variables are required:
```

Comment:
There are lot many variables to configure, I just wanted to introduce the section so that, I could include some notes on max file size.